packages/components/src/__future__/Tag/RemovableTag/subcomponents/RemoveButton.module.scss

```diff
@@ -0,0 +1,96 @@
+@import "~@kaizen/design-tokens/sass/border";
+@import "~@kaizen/design-tokens/sass/color";
+@import "../../../../../styles/button-reset";
+
+.removeButton {
+  @include button-reset;
+
+  position: relative;
+  display: inline-flex;
+  border-radius: 50%;
+
+  &:focus {
+    outline: none;
+  }
+
+  &:focus-visible {
+    &::after {
+      $focus-ring-offset: -3px;
+
+      border-color: $color-blue-500;
+      content: "";
+      position: absolute;
+      background: transparent;
+      border-width: $border-focus-ring-border-width;
+      border-style: $border-focus-ring-border-style;
+      border-radius: 50%;
+      inset: $focus-ring-offset;
+    }
+  }
+
+  &::before {
+    position: absolute;
+    content: "";
+    inset: -5px;
+    border-radius: 50%;
+  }
+
+  .closeIcon {
+    fill: rgba($color-purple-800-rgb, 0.7);
+
+    &:hover,
+    &:focus-visible {
+      cursor: pointer;
+      fill: $color-purple-800;
+    }
+  }
```

Comment:
Probably remove this from nesting as well, given we use CSS Modules it's unnecessary to introduce more CSS to enforce a child selector.